app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -305,9 +305,18 @@ protected void onDestroy() {
         super.onDestroy();
         // unregister our receiver
         this.unregisterReceiver(this.mReceiver);
+        KissApplication.getCameraHandler().releaseCamera();
```

Comment:
no, i mean:

on most case, KissApplication.getCameraHandler() will still be null at this point.

Therefore, there is no real need to create a new CameraHandler (which is what `getCameraHandler` does when the instance is null) to release the camera right after (`releaseCamera`)